File: proxy/tests/script_engine/url_password.lua
pcall(require, "../../script_libs/lua/roxy.lua")
---@type Extension
local url_password = {
	request = function(flow)
		if flow.request.url.password == "1234" then
			flow.request.url.password = "abcd"
		end
	end,
}
Extensions = { url_password }

File: proxy/tests/script_engine/url_password.js
/// <reference path="../../script_libs/js/roxy.d.ts" />
/** @type {Extension} */
const url_password = {
  request(flow) {
    if (flow.request.url.password == "1234") {
      flow.request.url.password = "abcd";
    }
  },
};
globalThis.extensions = [url_password]
